### ISSUE TITLE
interval name is not standard

### ISSUE THREAD

**Issue by sugizo** [mikehadlow/gtr-cof#47]
the 8 semitone : 
in whole tone scale is written as A5 stand for augmented fifth
in diatonic is written as m6 stand for minor sixth

yet the semitone is same, but the name is not standard, perhaps you can have a button to choose the interval name
Minor, major, or perfect intervals
and
Augmented or diminished intervals

ref:
https://en.wikipedia.org/wiki/Interval_(music)#Main_intervals

thanks

**Comment by mikehadlow** [comment]
So guitar dashboard is driven by an algorithm that calculates the names of notes and intervals based on a formula that works quite well for 7 note scales, but falls down on scales with different number of notes. I think this would require some thought and changes to the algorithm, so not something that will happen very quickly I'm afraid.

**Comment by sugizo** [comment]
seems misunderstood about this, all about the semitone naming either using : 
Minor, major, or perfect intervals
or
Augmented or diminished intervals

the reason raised this issue
as you can see 
Ab (A Flat) naming : 
- in diatonic is written as m6 stand for minor sixth
- in whole tone scale is written as A5 stand for augmented fifth

![screen_shot 1](https://user-images.githubusercontent.com/2418295/92377095-d0f8a580-f12d-11ea-83a3-168ad685930c.png)
![screen_shot](https://user-images.githubusercontent.com/2418295/92377104-d524c300-f12d-11ea-914e-593ea60463c6.png)

thanks

**Comment by mikehadlow** [comment]
OK, I see what you mean. So for a whole tone scale in C you'd expect to see:
C-1 D-M2 E-M3 Gb-A4 **Ab-m6** **Bb-m7** ?

**Comment by sugizo** [comment]
yeah, btw, your current work is not wrong, just nice to have a button to select between the interval name
Minor, major, or perfect intervals
or
Augmented or diminished intervals

some learn interval name using minor, major or perfect interval name, but some usually use augmented or diminished intervals name

perhaps this button value can be store in cookies too (human behaviour is not always same each other)

thanks
